static/js/components/SiteSidebar.tsx

```diff
@@ -1,45 +1,99 @@
 import * as React from "react"
 import { NavLink } from "react-router-dom"
 
-import { siteCollaboratorsUrl, siteContentListingUrl } from "../lib/urls"
+import { siteContentListingUrl } from "../lib/urls"
 
 import { ConfigItem, Website } from "../types/websites"
 
+interface MenuLink {
+  label: string
+  name: string
+}
+
+type NavMenuItem = ConfigItem | MenuLink
+
 interface Props {
   website: Website
 }
-export default function SiteSidebar(props: Props): JSX.Element {
-  const { website } = props
 
-  const configItems: ConfigItem[] = website.starter?.config?.collections ?? []
+const collaboratorMenuItem: MenuLink = {
+  name:  "collaborators",
+  label: "Collaborators"
+}
+
+/*
+ * This function adds a "Settings" menu header if it doesn't exist, then adds a link to the Collaborators page
+ */
+const addCollaboratorMenuItem = (
```

Comment:
It looks like this function and the `forEach` below do very similar things. Maybe this should be a utility function:

    const addToList = (map: Map<string, Array<NavMenuItem>, key: string, value: NavMenuItem) => {
        const list = map.get(key) ?? []
        map.set(key, [...list, value])
    }

And then below:

    const configMap = new Map<string, Array<ConfigItem>>()
    configItems.forEach((configItem: ConfigItem) => {
      addToList(configMap, configItem.category, configItem)
    })
    if (website.is_admin) {
      addToList(configMap, "Settings", collaboratorMenuItem)
    }

What do you think?
    }